src/Umbraco.Web/Models/Culture.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Umbraco.Web.Models
+{
+    public class Culture
```

Comment:
This is in the wrong namespace, models for editing in the back office go in `Umbraco.Web.Models.ContentEditing` and generally are mixed between `Display` and `Save` models if necessary, same goes for the Language model

Reply:
(I've changed this now)